ACP2E-460: [Documentation] Only images uploaded within the product description are managed by the new media gallery (#1676)
magento/merchdocs: src/cms/media-gallery.md

![Images displayed in the Media Gallery grid]({% link cms/assets/media-gallery-grid.png %}){: .zoom}
_Media Gallery grid_

{:.bs-callout-info}

> Product Gallery images from the `Images and Videos` product section are not managed by the New Media Gallery.
> Only images, which are used in the `Content` product section fields, are displayed and filtered in the New Media Gallery.

Reviewer: We have a topic for that type, so let's add the link in this note. 

```suggestion
Product Gallery images added to the [_Images and Videos_ product section]({% linkcatalog/product-image-upload.md %}) are not managed by the New Media Gallery. Only images used in the _Content_ product section fields are displayed and filtered in the New Media Gallery.
```

Author: Done
